c2corg_ui/static/js/addassociation.js

```diff
@@ -65,11 +83,23 @@ app.AddAssociationController = function(appApi, $attrs) {
 
 
 /**
+ * Associate two documents. Possible associations are listed under http://bit.ly/1T15jxx
```

Comment:
Do we really need a bit.ly link here? Why not directly using the underlying github URL?